save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/imageeditor/ReactAvatarImageCropper.kt

```diff
@@ -0,0 +1,26 @@
+@file:Suppress("HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE", "FILE_NAME_MATCH_CLASS")
+@file:JsModule("react-avatar-image-cropper")
+@file:JsNonModule
+
+package com.saveourtool.save.frontend.externals.imageeditor
+
+import org.w3c.dom.events.Event
+import react.*
+import web.file.File
+
+/**
+ * External declaration of [reactAvatarImageCropper] react component
+ */
+@JsName("default")
+external val reactAvatarImageCropper: FC<ReactAvatarImageCropperProps>
+
+/**
+ * Props of [ReactAvatarImageCropperProps]
+ */
+@Suppress("TYPE_ALIAS")
+external interface ReactAvatarImageCropperProps : PropsWithChildren {
```

Comment:
Why `PropsWithChildren` and not just `Props`?

On the call site, I don't see `reactAvatarImageCropper` having any children.

Reply:
Fixed